lib/working_hours/config.rb

```diff
@@ -151,6 +178,21 @@ def validate_working_hours! week
         end
       end
 
+      def validate_working_hours! week
+        if week.empty?
+          raise InvalidConfiguration.new "No working hours given"
+        end
+        if (invalid_keys = (week.keys - DAYS_OF_WEEK)).any?
+          raise InvalidConfiguration.new "Invalid day identifier(s): #{invalid_keys.join(', ')} - must be 3 letter symbols"
+        end
+      end
+
+      def validate_holiday_hours! days
+        if (invalid_keys = (days.keys.select{ |day| day.class.to_s != 'Date' })).any?
```

Comment:
A bit more idiomatic ruby IMO:
```suggestion
        if (invalid_keys = (days.keys.reject { |day| day.is_a?(Date) })).any?
```

Reply:
Yep, much nicer, fixed